analysis_runner/__init__.py

```diff
@@ -1,93 +0,0 @@
-"""analysis_runner module initialization."""
```

Comment:
I'm a bit torn: it's not great to break existing workflows, but on the other hand we really want to move the existing uses of `output_path` over to `cpg-utils` (and remove the dependencies on `analysis-runner`), which is difficult when there's no clear signal (such as the workflow breaking otherwise).

For backwards compatibility we'd also have to rename the `dataset_path` import to `bucket_path`, but that is of course easily doable.